EvidenceApi.Tests/V1/Factories/DomainFactoryTests.cs

```diff
@@ -18,6 +18,25 @@ public class DomainFactoryTests
     {
         private Fixture _fixture = new Fixture();
 
+        [Test]
+        public void CanMapEvidenceRequestEntityToDomainObject()
+        {
+            var entity = _fixture.Build<EvidenceRequestEntity>()
+                .Without(x => x.Communications)
+                .With(x => x.DeliveryMethods, new List<string> { "EMAIL" })
```

Comment:
You might need to add `Without(x => x.DocumentSubmissions)` or it will fail once I merge the PR for DocumentSubmission. I had to update the previous tests that were creating an `EvidenceRequestEntity` with ☝️

Reply:
Yeah I noticed. I feel like it might be useful to create a helper for that, so we don't have to keep finding places to fix things like that, WDYT?